### DISCUSSION TITLE
v1.1 is out!

### DISCUSSION THREAD

**Discussion by ShafSpecs** [ShafSpecs/remix-pwa#58] [Announcements]
## v1.1.0 is finally out!

A while after the release of v1.0, we have finally bundled all the latest changes and shipped right to your doorstep!

**Changelog:**
- Added package manager auto-detection; `remix-pwa` now detects and changes accordingly.
- Added CLI flag commands. Run `npx remix-pwa@latest -h` to find out :wink:. Thank you @dev-afzalansari
- Fixed Service Worker registration in production environment
- Little patches applied to Push API handler in Service Worker file
- Cleaned up the CLI code; it should be more readable & maintainable now :sweat_smile: 
- Added typings to file
- Expansion for Push API capabilities in future updates
- Minor bug fixes :hammer_and_wrench: 

<hr /><em>This discussion was created from the release <a href='https://github.com/ShafSpecs/remix-pwa/releases/tag/v1.1'>v1.1 is out!</a>.</em>

**Comment by carlosvirreira** [comment]
This is so awesome, thank you for creating this! We will be using it for https://github.com/Shelf-nu/shelf.nu

**Reply by ShafSpecs** [reply]
Awesome! Seeing ppl using it brings us joy 🥳. An even bigger update is around the corner
